python/py-argh/Portfile

```diff
@@ -21,11 +21,28 @@ checksums           rmd160  42da2aa1dfae654419acfc62db73d0fb27ecd23f \
                     sha256  e9535b8c84dc9571a48999094fda7f33e63c3f1b74f3e5f3ac0105a58405bb65 \
                     size    32913
 
-python.versions     27 35 36 37 38 39
+python.versions     27 35 36 37 38 39 310
```

Comment:
Presumably we want to get rid of 35/36 here?

Reply:
I agree. 36 is past end of life. If you remove this, then you can also remove the `if {${python.version} >= 37}` conditional.